website/sidebars.json

```diff
@@ -0,0 +1,35 @@
+{
+  "docs": {
+    "Users": ["users/installation", "users/configuration", "users/faq"],
+    "Rule Authors": [
+      "rule-authors/setup",
+      "rule-authors/tutorial",
+      "rule-authors/glossary"
+    ]
+  },
+  "rules": {
```

Comment:
Can we generate this?

Reply:
We can, but it's simplest to keep it manual for now. Ideally, I want `docs.Main` to take care of generating data like this and also start the live reload server.